package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ember-async-component",
-  "version": "0.0.3",
+  "version": "1.0.0",
```

Comment:
I would say 0.1.0 or 0.2.0 to let it bake for longer, although we've already got some miles out of it.

Reply:
😆 won't run out of integers 😛

Reply:
I think this is production ready. So I pushed it to 1.x.x 
Any strong opinion against it?